orgs-using-d.dd

```diff
@@ -107,15 +119,16 @@ $(P
 )
 
 Macros:
-	TITLE=Organizations using the D Language
-	FA_ICON=<i class="fa fa-$1" aria-hidden="true"></i>
-	FA_HIRING=$(FA_ICON rocket) $(WEB $1, hiring)
-	FA_GITHUB=$(FA_ICON github) $(HTTPS github.com/$1, Github)
-	FA_TALK=$(FA_ICON youtube-play) $(WEB $1, $2)
-	FA_SEPARATOR=&nbsp;
-	FA_TESTIMONIAL=$(FA_ICON heart)
-	LINK_ROW=$(DIVC orgs-using-d-linkrow, $1)
-	DORGIMG=<a class="orgs-using-d-img-wrapper vcontainer-box" href="$1"><img class="vcontainer-element" src="$(ROOT_DIR)images/orgs-using-d/$2" /></a>
+    TITLE=Organizations using the D Language
+    FA_ICON=<i class="fa fa-$1" aria-hidden="true"></i>
+    FA_HIRING=$(FA_ICON rocket) $(WEB $1, Hiring)
+    FA_GITHUB=$(FA_ICON github) $(HTTPS github.com/$1, Github)
+    FA_TALK=$(FA_ICON youtube-play) $(WEB $1, $2)
+    FA_SEPARATOR=&nbsp;
+    FA_TESTIMONIAL=$(FA_ICON heart)
+    FA_QUOTE=$(FA_ICON quote-left) $1 $(FA_ICON quote-right)
```

Comment:
I think if you use `$0` instead of `$1` here, you won't need to escape commas with `$(COMMA)` when using this macro.